Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -1982,10 +1116,16 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
 
   for (size_t i = bank0; i < bankn; i++) {
     // We make tasks for loading
-    if (bankNumEvents[i] > 0)
-      pool.schedule(new LoadBankFromDiskTask(
+    if (bankNumEvents[i] > 0) {
+      LoadBankFromDiskTask *task = new LoadBankFromDiskTask(
```

Comment:
Since the mutex(es) prevents things from reading in parallel this runs in series anyhow and complicates the code. [The tests](https://github.com/OwenArnold/hdf5_vs_nexus/blob/master/read_results.md) showed that using HDF's enumeration mechanism is significantly faster for the cases we are looking at.